frontend/packages/metal3-plugin/src/features.ts

```diff
@@ -0,0 +1,19 @@
+import * as _ from 'lodash';
+import { k8sGet, K8sResourceKind } from '@console/internal/module/k8s';
+import { InfrastructureModel } from '@console/internal/models';
+import { getInfrastructurePlatform } from '@console/shared/src/selectors/infrastructure';
+import { setFlag, handleError } from '@console/internal/actions/features';
+import { ActionFeatureFlagDetector } from '@console/plugin-sdk';
+
+export const BAREMETAL = 'BAREMETAL';
```

Comment:
For consistency, I'd change this to

```ts
export const BAREMETAL_FLAG = 'BAREMETAL';
```

since it's not part of `FLAGS` enum anymore.